heroku/cli
# chore(auth): add code coverage reports

## Why the change?
[GUS Card](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001NBzQoYAL/view)

As per our review of unit tests and coverage for the auth package, this PR does the following:
- Ensures that our code coverage reports over 85% test coverage of statements ✅


## Additional Notes
- Removed `all` property in `.nycrc` file to remove `src/index.ts` from testing since there is nothing to test in that file
- Removed unreachable code in [2fa/disable.ts](https://github.com/heroku/cli/pull/2348/files#diff-0bd3e95dc3103687accddb9d929fa2b75ac27fb6f77b0d3f96e4bf61b74a6387)
- `login.ts` should be ignored as it is already tested in the [@heroku-cli/command repo](https://github.com/heroku/heroku-cli-command/blob/master/test/login.test.ts). This should be the case for `logout.ts`, but I already added a test for it. Couldn't hurt in coverage!

## How to verify?
1. Pull down branch
2. Run tests
3. Confirm test coverage report is visible and that it reports over 85% test coverage of statements for all files except for `login.ts`
